src/toil_scripts/adam_variants/convert_dbsnp.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -x -v
+
+python -m toil_scripts.adam_variants.convert_vcf \
+    aws:us-west-2:FIXME-toil-jobstore \
+    --num_nodes 3 \
```

Comment:
Not sure how many nodes & what memory settings will work best for a file of this size